automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/ProfilePage.scala

```diff
@@ -29,4 +29,14 @@ class ProfilePage(implicit webDriver: WebDriver) extends BaseFireCloudPage
     }
     readText(proxyGroupEmailQuery)
   }
+
+  def awaitProvidersReady: Unit = {
+    await notVisible (cssSelector("[data-test-id=spinner]"), 60)
+  }
+
+  def linkProvider(provider: String): Unit = {
```

Comment:
consider renaming to something more verb like.  Maybe `clickProviderLink` or something